### PR TITLE
Mobile first portfolio pull request

### PR DESCRIPTION
**Pull request by kkasaija**
Set up a header and section outline

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
-# mobile_first_portfolio
-Create a portfolio "mobile first" project
+# Build mobile_first_portfolio project
+
+Project objectives
```
Comment (MahmoudBakr23) [inline README.md]: **Optional**
- Please follow the great [template](https://github.com/microverseinc/readme-template) from Microverse to build a more descriptive README file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+<body>
+  <main class="main">
+    <div class="container">
+      <header>
+        <div class="logo"><a href="index.html">My Logo</a></div>
+        <nav>
+          <a href=""><img id="dropdown" src="images/Union.svg" alt="menu" /></a>
+        </nav>
+      </header>
+      <div class="content">
```
Comment (MahmoudBakr23) [inline index.html:22]: - [ ] Please make sure to follow the Figma design template and attach the background image as in the original design

**Your design**
Screenshot (1)

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Kasaija-Kenneth 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done with following the original design so far
- Great work with semantic HTML
- Descriptive PR
- Correct Linters

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please see the comment under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**